### ISSUE TITLE
DMN Diagrams from C7 Modeler are not deployable

### ISSUE THREAD

**Issue by hauptmedia** [operaton/operaton#194]
**Describe the bug**
DMN Diagrams from C7 Modeler are not deployable

**To Reproduce**
1. Create an empty DMN Diagram in the Modeler
2. Set TTL
3. Try to deploy
4. Get Error: ENGINE-22004 Unable to transform DMN resource 'diagram_1.dmn'. [ deploy-error ]

**Expected behavior**
DMN Diagrams should be deployable like BPMN Diagrams

**Comment by javahippie** [comment]
Looking into this, thanks for reporting!

**Comment by hauptmedia** [comment]
Perhaps it has something to do with this here? https://github.com/operaton/operaton/blob/9cc4fd7148be00920acc4a063afc5fdda91160d1/model-api/dmn-model/src/main/java/org/operaton/bpm/model/dmn/impl/DmnModelConstants.java#L58

**Comment by javahippie** [comment]
Yes, that's what I'm suspecting, too. For the BPMN model there is a fallback mechanism, accepting `camunda` namespaces, too. This is missing in the DMN, it should be possible to port this

**Comment by hauptmedia** [comment]
It's working btw with the operaton xml namespace

**Comment by javahippie** [comment]
I'm wondering if something similar should be done for CMMN, which is still a part of the engine

**Comment by hauptmedia** [comment]
Perhaps we should start another discussion in the forum if anyone is actually using CMMN with C7. I think we should just drop the support for it.
